modules/gdnavigation/rvo_agent.cpp

```diff
@@ -80,5 +80,7 @@ void RvoAgent::dispatch_callback() {
 
 	const Variant *vp[2] = { &callback.new_velocity, &callback.udata };
 	int argc = (callback.udata.get_type() == Variant::NIL) ? 1 : 2;
-	obj->call(callback.method, vp, argc, responseCallError);
+	if (obj) {
```

Comment:
I think this should be an early return in the previous `if (obj == nullptr)` instead.
@AndreaCatania

Reply:
Yes, it should be an early return on the above `if`.